tutorials/i18n/internationalizing_games.rst

```diff
@@ -68,6 +68,28 @@ will automatically fetch a translation if their text matches a translation key.
 For example, if a label's text is "MAIN_SCREEN_GREETING1" and that key exists
 in the current translation, then the text will automatically be translated.
 
+This automatic translation behavior may be undesirable in certain cases. For
+instance, when using a Label to display a player's name, you most likely don't
+want the player's name to be translated if it matches a translation key. To
+disable automatic translation on a specific node, use the following code
+snippet::
```

Comment:
Would be good to reference the `set_message_translation` classref in this paragraph (especially for users of other langs than GDScript).